src/main/java/de/uni_tuebingen/ub/nppm/db/SaveHelper.java

```diff
@@ -47,6 +50,57 @@ public static List<Map<String, String>> selectAttribute(String table, String att
         return results;
     }
 
+    public static List<Map> selectAttributeMap(String table, String attributeString, int id) throws Exception {
+        List<Map> results = new ArrayList<>();
+
+        try ( Session session = getSession()) {
+            session.getTransaction().begin();
+            String sql = "SELECT " + attributeString + " FROM " + table + " WHERE ID = " + id; // SQL-Abfrage erstellen
+
+            NativeQuery query = session.createNativeQuery(sql);
```

Comment:
If you want to execute a plain SQL statement & get the results as `List<Object[]>,` you can use the parent's function `AbstractBase.getListNative(String sql)`. Then you have to write less code, you don't even have open a session manually for this.

Reply:
done:  look at commit: use of AbstractBase.getListNative